bentoml/_internal/frameworks/tensorflow_v2.py

```diff
@@ -377,8 +377,10 @@ def from_batch_payloads(
         return cls.batches_to_batch(batches, batch_dim)
 
 
-DataContainerRegistry.register_container(
-    LazyType("tensorflow.python.framework.ops", "_EagerTensorBase"),
-    LazyType("tensorflow.python.framework.ops", "_EagerTensorBase"),
-    TensorflowTensorContainer,
-)
+# NOTE: we only register the TensorflowTensorContainer as if eager execution is enabled.
+if tf.executing_eagerly():
```

Comment:
https://github.com/tensorflow/tensorflow/blob/d8ce9f9c301d021a69953134185ab728c1c248d3/tensorflow/python/eager/context.py#L2162 @larme

Reply:
@aarnphm  then should we just change `_EagerTensorBase` to `Tensor` here?

Reply:
i had that in mind as well